src/corelibs/U2Lang/src/support/SimpleWorkflowTask.cpp

```diff
@@ -148,6 +149,14 @@ SimpleMSAWorkflow4GObjectTask::SimpleMSAWorkflow4GObjectTask(const QString &task
     MultipleSequenceAlignment al = MSAUtils::createCopyWithIndexedRowNames(msaObjectPointer->getMultipleAlignment());
 
     MultipleSequenceAlignmentObject *msaObject = MultipleSequenceAlignmentImporter::createAlignment(msaObjectPointer->getEntityRef().dbiRef, al, os);
+    for (const MultipleSequenceAlignmentRow &row : al->getMsaRows()) {
+        if (row->getData().contains("*")) {
+            msaObject->replaceAllCharacters('*', 'X');
+            ioLog.info(tr("Symbols '*' has been replaced with 'X' during alignment."));
```

Comment:
Any MSA with '*' char has RAW alphabet. It means that it may also have any other character: braces, &, > < ! etc...
Did you check that KAlign works with all of them, but only fails with '*'?